### DISCUSSION TITLE
Education email cannot receive verification email from Github

### DISCUSSION THREAD

**Discussion by xiaoming-ouc** [community/community#145914] [GitHub Education]
### Hi everyone,

Education email cannot receive verification email from Github. When I sent an email to *.stu.ouc.edu.cn, I still could not receive the verification email from Github in my education mailbox, so that I could not complete the education verification.
I wonder if the Github staff can help me with this.

**Comment by Maksimonk** [comment]
Hi. Have you tried these options?

1) Check the spam folder (even emails from GitHub can get there).
2) Check that the address is correct.
3) Try to resend the email (maybe even several times).
4) Try contacting your mail administrators (there is a possibility that they block all such emails).

I applied for a PRO by student edu mail. There were no problems receiving the letter at all.

**Comment by GabJuarez** [comment]
Check your Spam/Junk folder: Sometimes, emails from GitHub might end up there.

Verify your email address: Double-check that the email you entered in your GitHub account is correct, and that it’s the one associated with your education institution.

Email Blocking/Filtering: Some educational email providers might have strict filters in place. You can reach out to your institution’s IT support to see if they’re blocking GitHub emails.

Wait and Try Again: There might be a delay in receiving the email. Sometimes it takes a while for the verification email to come through.

If none of these work, you can contact GitHub support directly. They can help you resolve issues with verification emails and ensure you're able to complete the Education verification process. Here's the link to contact GitHub support: [GitHub Support](https://support.github.com/contact).

Good luck, and hopefully, it gets sorted soon!
